editors/vscode/src/preview-compat.ts

```diff
@@ -14,7 +14,18 @@ import {
 } from "./preview";
 import { tinymist } from "./lsp";
 import { loadHTMLFile } from "./util";
+import {
```

Comment:
I don't want to add new feature in `preview-compat.ts`, which may cause problem. People who want to use new features should move to tinymist.

Reply:
Reverted. Thank you for your review.